### PR TITLE
upgrade react router dom

### LINKED ISSUE #223: Upgrade React Router Dom
We need to upgrade React Router Dom to allow upgrades to React, React Dom and MUI. #219 

https://reactrouter.com/docs/en/v6/upgrading/v5

### PR DESCRIPTION
**Pull request by laitanop**
1.  Upgrading from "react-router-dom": "^5.2.0",   To  "react-router-dom": "^6.2.1"
2. Replaced from  "Switch " to "Routes"
3. Replaced from  "Redirect " to "Navigate"
4.  Replaced from component={} to  element={</>}
5. Replaced from  "useHistory " to "useNavigate"

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -29,63 +29,57 @@ const App = () => {
...
-                            exact
                             path="/network/:network/query"
-                            component={Query}
+                            element={<Query />}
                         />
                         <Route
-                            exact
                             path="/network/:network/query/:groupingId/group/:groups"
-                            component={Query}
+                            element={Query}
```
Comment (phillip055) [inline src/App.tsx]: does this need to be <Query /> too?

**Review by phillip055 — CHANGES_REQUESTED** [review]
Other than that, LGTM

**Comment by magikadev** [comment]
@laitanop Would you please rebase with master branch?

**Comment by magikadev** [comment]
https://yearn-watch-a2667hw10-y-finance.vercel.app/network/ethereum/query
https://yearn-watch-a2667hw10-y-finance.vercel.app/network/ethereum/report
https://yearn-watch-a2667hw10-y-finance.vercel.app/network/ethereum/risk
The above routes are not working.

**Pull request merged**